.github/workflows/nodejs.yml

```diff
@@ -5,8 +5,9 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ develop, master, 2.0 ]
+    branches: [develop, master, 2.0]
   pull_request:
+    branches: [develop, 2.0]
```

Comment:
Again, I disagree with this - I want tests to run on all PRs, regardless of their parent branches. So if you have branches chaining, you can still get tests on child branches.

Reply:
Ah I had to re-add that to get CI to run on my branch